storage/src/main/java/org/apache/uniffle/storage/common/DefaultStorageMediaProvider.java

```diff
@@ -84,4 +83,12 @@ public StorageMedia getStorageMediaFor(String baseDir) {
     logger.info("Default storage type provider returns HDD by default");
     return StorageMedia.HDD;
   }
+
+  @VisibleForTesting
+  String getDeviceName(String mountPoint) {
```

Comment:
Thanks @xianjingfeng for update the PR. Can it be static?

Reply:
Done